chunky/src/java/se/llbit/chunky/renderer/postprecessing/ACESFilmicFilter.java

```diff
@@ -0,0 +1,32 @@
+package se.llbit.chunky.renderer.postprecessing;
+
+import org.apache.commons.math3.util.FastMath;
+import se.llbit.chunky.renderer.scene.Scene;
+import se.llbit.math.QuickMath;
+
+public class ACESFilmicFilter extends IndependentPostProcessingFilter {
+  private static final float aces_a = 2.51f;
+  private static final float aces_b = 0.03f;
+  private static final float aces_c = 2.43f;
+  private static final float aces_d = 0.59f;
+  private static final float aces_e = 0.14f;
+  
+  @Override
+  public void processPixel(double[] pixel) {
+    // https://knarkowicz.wordpress.com/2016/01/06/aces-filmic-tone-mapping-curve/
```

Comment:
Maybe `@link` this in the class javadoc? That way the source will show up in the javadocs.